js/MVP/components/VeilederSpeechBubble.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
 function VeilederSpeechBubble({ content }: Props): ReactElement {
   return (
     <VeilederStyled tekst={content} posisjon="høyre" storrelse="S" fargetema="info" hvitSnakkeboble>
-      <VeilederAvatar />
+      <img src={VeilederAvatar} />
```

Comment:
Jeg lagde en liten greie i Dialogmøte Sykmeldt for å gjøre det lettere å bruke bilder, og refaktorere. Du kan jo vurdere om du vil gjøre det her også: https://github.com/navikt/dialogmote/blob/master/js/images/imageComponents.ts